about.html

```diff
@@ -1,68 +1,97 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
-</head>
+</head>  
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+    <div class="container-about-page">
+      <div class="header">
+        <div class="logo">
+          <img src='img/lambda-black.png'>
+      </div>  
+        <nav>
+          <div>
+            <a class='home' href="index.html">Home</a>
+          </div>
+          <div>
+            <a class='about' href="about.html">About</a>
+          </div>
+          <div>
+            <a class='products' href="#">Products</a>
+          </div>
+          <div>
+              <a class='blog' href="#">Blog</a>
+            </div>
+            <div>
+              <a class='contact' href="#">Contact</a>
+            </div>
+        </nav>
+        </div>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <section class="backgroundImage">
+          <img src='img/jumbo-about.png'>
+      </section>
 
-    Learn More
+    <section class="aboutText">
+    <h2>About</h2>
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </section>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+  <section class="thumbnails">
```

Comment:
You done a good job of adding Semantic HTML, please be mindful of indentation and removing unnecessary spaces to aid in code readability. is a good habit to start building